app/views/shelters/show.html.erb

```diff
@@ -15,7 +24,9 @@
       <p><span>State:</span> <%= @shelter.state %></p>
       <p><span>Zip:</span> <%= @shelter.zip %></p>
       <p><span>Pet count:</span> <%= @shelter.pet_count %></p>
-      <p><span>Average rating:</span> <%= @shelter.average_rating %></p>
+      <% if !@shelter.reviews.empty? %>
```

Comment:
Thanks for putting in this addition. I didn't even think about if there were no reviews for a shelter...